pkgs/applications/networking/mullvad/mullvad.nix

```diff
@@ -115,13 +115,12 @@ rustPlatform.buildRustPackage rec {
 
   passthru = {
     inherit libwg;
-    inherit openvpn-mullvad;
   };
 
   meta = {
     description = "Mullvad VPN command-line client tools";
     homepage = "https://github.com/mullvad/mullvadvpn-app";
-    changelog = "https://github.com/mullvad/mullvadvpn-app/blob/2025.2/CHANGELOG.md";
+    changelog = "https://github.com/mullvad/mullvadvpn-app/blob/${version}/CHANGELOG.md";
```

Comment:
(No need to change it back, but the reason I started hardcoding the version -- despite forgetting to ever update it lol -- was so that I could click / copy-paste the link and not have to remember what version I wanted)